internal/oci/oci_linux.go

```diff
@@ -50,6 +53,10 @@ func (r *runtimeOCI) containerStats(ctr *Container, cgroup string) (*ContainerSt
 	stats.Container = ctr.ID()
 	stats.SystemNano = time.Now().UnixNano()
 
+	if ctr.Spoofed() {
```

Comment:
Not a change request but a thought that we shouldn't be queried about container stats from the kubelet.